prox_config_backup.sh

```diff
@@ -1,32 +1,55 @@
 #!/bin/bash
-# Version	      0.2.2 - BETA ! !
-# Date		      02.20.2020
+# Version	      0.2.3 - BETA !!
+# Date		      04.17.2020
 # Author 	      DerDanilo 
-# Contributors    aboutte, xmirakulix, bootsie123
+# Contributors    aboutte, xmirakulix, bootsie123, phidauex
 
-# set vars
+###########################
+# Configuration Variables #
+###########################
 
-# always exit on error
-set -e
-
-# permanent backups directory
-# default value can be overridden by setting environment variable before running prox_config_backup.sh
-# example: export BACK_DIR="/mnt/pve/media/backup"
-# or
-# example: BACK_DIR="." ./prox_config_backup.sh
-_bdir=${BACK_DIR:-/mnt/backups/proxmox}
+# Permanent backups directory
+# Default value can be overridden by setting environment variable before running prox_config_backup.sh
+#   example: export BACK_DIR="/mnt/pve/media/backup"
+#   or
+#   example: BACK_DIR="." ./prox_config_backup.sh
+DEFAULT_DIR="/mnt/pve/media/backup"
```

Comment:
Please rename to `DEFAULT_BACK_DIR` (and all references of course)

Reply:
Updated, good call to make the variable names more consistent.